File: PYTHON/Questões Uri/1043.py
a,b,c=map(float, input().split(" "))

if (a+b)<=c or (b+c)<=a or (a+c)<=b or (a+b)<=0:
  area=(((a+b)*c)/2)
  print("Area = %0.1f"%area)

elif (a+b)>c or (b+c)>a or (a+c)>b:
  perimetro=(a+b+c)
  print("Perimetro = %0.1f"%perimetro)

File: JAVASCRIPT/Questões uri/1043.js
const pegarValoress = (line) => line.split(" ").map(a => +a)
const [a,b,c] = pegarValoress(lines.shift())


if (((a+b)<=c) || ((b+c)<=a) || ((a+c)<=b) || ((a+b)<=0)){
    let area=(((a+b)*c)/2)
    console.log(`Area = ${area.toFixed(1)}`)
}else if (((a+b)>c) || ((b+c)>a) || ((a+c)>b)){
    let perimetro=(a+b+c)
    console.log(`Perimetro = ${perimetro.toFixed(1)}`)
}